_data/simple-icons.json

```diff
@@ -1446,7 +1446,7 @@
         },
         {
             "title": "Aston Martin",
-            "hex": "000000",
+            "hex": "161A11",
```

Comment:
```suggestion
            "hex": "00665E",
```

Reply:
This welcome [page](https://www.astonmartin.com/en/welcome) bg img uses green too.

Reply:
It seems we have consensus, so 🙂 Will update the colour and get this merged.